docs/source/advanced/http-engine.mdx

```diff
@@ -1,20 +1,21 @@
 ---
-title: Use your own HTTP client
+title: Using a custom HTTP library
+sidebar_title: Custom HTTP libraries
```

Comment:
I'd keep the "HTTP client" terminology as it's been widely popularized by "OkHttpClient"  and this feels more specific than "library"